### ISSUE TITLE
Live system x86_64 fails to install: "/mnt not found", "acces denied", "permission denied", "/sda1 nit found"

### ISSUE THREAD

**Issue by 0xBADBAC0N** [BlackArch/blackarch#1297]
Hi guys,
what i did:
- wrote the blackarch image to an usb drive (full iso and only netinstaller, tested both)
- started (worked fine so far)
- set up the ssd(create boot/root/home partition with ext4, made boot bootable)
- if the installer now try to start the installation he throw hundrets of errors like "sda1 not found"(even if its if i check in an other console there), "acces/permission denied"(even if i run the installer script as in the guide as sudo)
- the installer script then try to finish(set up user and so on) what obv wont work because the system is not right set up(after restart nothing working exists)

My request:
More checks in the installer scripts to have more exit points if the script fail at some point and not tryhard to finish the installation over dozen of errors.

What i need? Help...i even set up a regular arch linux(use it also on an other machine) and tried to install it on top of it but i faced the same problems

**Comment by mikeybro** [comment]
It took me a few tries, just gotta keep going through the process until you
get everything right  dyslexia is said to affect 1 in 4 people...

**Comment by mikeybro** [comment]
another thing, I didn't make a home partition, as bad as that sounds. My
linux admin skills are not that high, despite all the Linux Foundation
Training Pubs I read and all the arch wiki pages I've visited thousands of
times. What happened one time with me was my root partition (20GB) was
filled up and my system crashed, My 100GB+ home partition was never used
and I was able to use my brain to solve the problem , so I just left the
home partition out.
